code/modules/research/rdconsole.dm

```diff
@@ -160,8 +160,7 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 		files.known_tech[T.id] = T
 	for(var/v in files.known_tech)
 		var/datum/tech/KT = files.known_tech[v]
-		if(KT.level < KT.max_level)
-			KT.level=KT.max_level
+		KT.level = 8
```

Comment:
Wouldn't it make more sense to actually set the `max_level`s to `8`? Not that it is used anywhere other than here.

Reply:
My original intent was to outright remove the var, but then **_Economy_** rework became a thing.